### ISSUE TITLE
Creating a Policy class

### ISSUE THREAD

**Issue by ADGEfficiency** [ADGEfficiency/energy_py#10]
In RL it's possible to use a variety of different policies - most common being epsilon-greedy.  This is currently hard coded into DQN - would be nice to pull it out and be able to use other methods (ie random, softmax, Boltzman)

See
https://medium.com/emergent-future/simple-reinforcement-learning-with-tensorflow-part-7-action-selection-strategies-for-exploration-d3a97b7cceaf

In the link below the policy is defined as a method
https://ewanlee.github.io/2017/07/09/Using-Tensorflow-and-Deep-Q-Network-Double-DQN-to-Play-Breakout/

**Comment by ADGEfficiency** [comment]
This has been done in the recent DQN rebuild (see energy_py/common/policies)
